package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -55,9 +59,9 @@ get_iso()
 {
     if [ -n "$_COS_INSTALL_ISO_URL" ]; then
         ISOMNT=$(mktemp -d -p /tmp cos.XXXXXXXX.isomnt)
-        TEMP_FILE=$(mktemp -p /tmp cos.XXXXXXXX.iso)
-        get_url ${_COS_INSTALL_ISO_URL} ${TEMP_FILE}
-        ISO_DEVICE=$(losetup --show -f $TEMP_FILE)
+        ISOTEMP=$(mktemp -p ${TARGET}/usr/local cos.XXXXXXXX.iso)
```

Comment:
Since is persistent storage, it would be nice to remove the temp file when everything is done.

Reply:
The file will be removed in `cleanup()` function.